cli/commands/library/unicodeprint.zig

```diff
@@ -0,0 +1,41 @@
+const std = @import("std");
```

Comment:
I think this belongs in `cli/util.zig` - we already import this in `cli/commands/init.zig`.